javascript/myscripts.js

```diff
@@ -0,0 +1,411 @@
+        displayItemNewCost: 'Rs. 32',
+        displayItemOldCost: 'Rs. 41',
+        displayItemAddButton: 'ADD',
+        classToBeAssigned: 'card all exotics'
+    },
+
+    {
+        displayItemOffer: '10% OFF',
+        displayItemImage: 'images/carrot-img.jpeg',
+        displayItemSource: 'Sourced at 5 AM',
+        displayItemTitle: 'Red Capsicum',
+        displayItemQuantity: '500 g',
+        displayItemNewCost: 'Rs. 17',
+        displayItemOldCost: 'Rs. 19',
+        displayItemAddButton: 'ADD',
+        classToBeAssigned: 'card all exotics'
+    },
+
+    {
+        displayItemOffer: '16% OFF',
+        displayItemImage: 'images/green-chilli-img.png',
+        displayItemSource: 'Sourced at 5 AM',
+        displayItemTitle: 'King Trumpet Mushroom',
+        displayItemQuantity: '100 g',
+        displayItemNewCost: 'Rs. 10',
+        displayItemOldCost: 'Rs. 12',
+        displayItemAddButton: 'ADD',
+        classToBeAssigned: 'card all exotics'
+    },
+
+
+    {
+        displayItemOffer: '11% OFF',
+        displayItemImage: 'images/potato-img.png',
+        displayItemSource: 'Sourced at 5 AM',
+        displayItemTitle: 'Amla',
+        displayItemQuantity: '1 Kg',
+        displayItemNewCost: 'Rs. 15',
+        displayItemOldCost: 'Rs. 17',
+        displayItemAddButton: 'ADD',
+        classToBeAssigned: 'card all seasonal'
+    },
+
+    {
+        displayItemOffer: '21% OFF',
+        displayItemImage: 'images/onion-img.png',
+        displayItemSource: 'Sourced at 5 AM',
+        displayItemTitle: 'Nagpur Orange',
+        displayItemQuantity: '1 Kg',
+        displayItemNewCost: 'Rs. 32',
+        displayItemOldCost: 'Rs. 41',
+        displayItemAddButton: 'ADD',
+        classToBeAssigned: 'card all seasonal'
+    },
+
+    {
+        displayItemOffer: '10% OFF',
+        displayItemImage: 'images/carrot-img.jpeg',
+        displayItemSource: 'Sourced at 5 AM',
+        displayItemTitle: 'Groundnut',
+        displayItemQuantity: '500 g',
+        displayItemNewCost: 'Rs. 17',
+        displayItemOldCost: 'Rs. 19',
+        displayItemAddButton: 'ADD',
+        classToBeAssigned: 'card all seasonal'
+    },
+
+
+    {
+        displayItemOffer: '11% OFF',
+        displayItemImage: 'images/potato-img.png',
+        displayItemSource: 'Sourced at 5 AM',
+        displayItemTitle: 'Organic Radish',
+        displayItemQuantity: '1 Kg',
+        displayItemNewCost: 'Rs. 15',
+        displayItemOldCost: 'Rs. 17',
+        displayItemAddButton: 'ADD',
+        classToBeAssigned: 'card all organicHydroponic'
+    },
+
+    {
+        displayItemOffer: '21% OFF',
+        displayItemImage: 'images/onion-img.png',
+        displayItemSource: 'Sourced at 5 AM',
+        displayItemTitle: 'Hydroponic Baby Corn',
+        displayItemQuantity: '1 Kg',
+        displayItemNewCost: 'Rs. 32',
+        displayItemOldCost: 'Rs. 41',
+        displayItemAddButton: 'ADD',
+        classToBeAssigned: 'card all organicHydroponic'
```

Comment:
1. Rather than using class names, you should maintain an array of categories in each item.
2. Create Enums for categories so that you do not have to maintain copies of string.